src/pages/FilesPage/FilesTabs/DatasourcesTable/DatasourcesTable.tsx

```diff
@@ -1,99 +1,100 @@
-import { Dropdown, Menu } from 'antd';
+import { Dropdown, Menu, TableColumnType } from 'antd';
 import React, { FC } from 'react';
 
 import { IDatasourceResponseObject } from '../../../../api/datasourcesService';
 import { ReactComponent as MenuArrowIcon } from '../../../../resources/menu-arrow.svg';
 import { ReactComponent as MoreIcon } from '../../../../resources/more.svg';
+import { Table } from '../../../../shared/Table/Table';
 import { ArticlesListItem } from '../../../SearchResultPage/SearchResultMain/ArticlesList/ArticlesListItem/ArticlesListItem';
 
 import style from './DatasourcesTable.module.scss';
 
-type DatasourcesProps = {
+const columns: TableColumnType<IDatasourceResponseObject>[] = [
+  {
+    dataIndex: 'article',
+    render: (article: IDatasourceResponseObject['article']) => (
+      <ArticlesListItem article={article} isMinimized={true} />
```

Comment:
Create duplicate of this component under current folder.
It was mistake to reuse this component in datasources table because dastasource and article are not the same.  
I missed it while checked previous PR.

Reply:
Done